README.md

```diff
@@ -27,544 +27,76 @@ Network Protocol parameters set on BMCs include:
 * NTP server 
 * Syslog server
 
```

Comment:
I would say all the changes Im seeing here should be in the swagger.  Are they?

Reply:
I think all the info is in the swagger, albeit more tersely stated.